ct/5etools.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: TheRealVira
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://5e.tools/
+
+# App Default Values
+APP="5etools"
+var_tags="wiki"
+var_cpu="1"
+var_ram="512"
+var_disk="10"
+var_os="debian"
+var_version="12"
+var_unprivileged="1"
+var_offline_mode="TRUE"
+
+# App Output & Base Settings
+header_info "$APP"
+base_settings
+
+# Core
+variables
+color
+catch_errors
+
+function update_script() {
+    header_info
+    check_container_storage
+    check_container_resources
+
+    # Check if installation is present | -f for file, -d for folder
+    if [[ ! -d "/opt/${APP}" ]]; then
+        msg_error "No ${APP} Installation Found!"
+        exit
+    fi
+
+    # Crawling the new version and checking whether an update is required
+    msg_info "Updating System"
+
+    apt-get update &>/dev/null
+    apt-get -y upgrade &>/dev/null
```

Comment:
```suggestion
    msg_info "Updating System"
    apt-get update &>/dev/null
    apt-get -y upgrade &>/dev/null
    msg_ok "Updated System"
```
Every msg_info needs a msg_ok or the spinner will get spwaned multiple times.